activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.java

```diff
@@ -54,14 +54,26 @@ public void testThrowErrorWithEmptyErrorCode() {
     }
   }
 
-  @Deployment
+  public void testThrowErrorWithoutErrorCode() {
+    try {
+      repositoryService.createDeployment().addClasspathResource("org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.testThrowErrorWithoutErrorCode.bpmn20.xml").deploy();
+      fail("ActivitiException expected");
+    } catch (ActivitiException re) {
+    }
+  }
+
   public void testCatchErrorOnEmbeddedSubprocessWithEmptyErrorCode() {
```

Comment:
Shouldn't this test keep initial `@Deployment` annotation here, so it keeps using the same process definition as before?